charts/garnet/templates/init-job.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if .Values.cluster.enabled }}
+{{- if .Values.cluster.initJob.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-manage-cluster"
+  labels:
+    {{- include "garnet.labels" . | nindent 4 }}
+  {{- with .Values.cluster.initJob.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  backoffLimit: {{ .Values.cluster.initJob.backoffLimit }}
+  ttlSecondsAfterFinished: {{ .Values.cluster.initJob.ttlSecondsAfterFinished }}
+  template:
+    metadata:
+      labels:
+        {{- include "garnet.labels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: init
+          image: "{{ .Values.cluster.initJob.image.registry }}/{{ .Values.cluster.initJob.image.repository }}:{{ .Values.cluster.initJob.image.tag | default "latest" }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: 
+            - /bin/sh
+            - -c
+            - |
+              echo "Waiting for DNS propagation..."
+              sleep 10
+
+              # Wait for Redis to be ready using ping
```

Comment:
nit; consider changing comments to reference Garnet instead

Reply:
Was this change already made?